pandas/tests/test_frame.py

```diff
@@ -12328,6 +12328,24 @@ def test_unstack_dtypes(self):
         expected = Series({'float64' : 2, 'object' : 2})
         assert_series_equal(result, expected)
 
+        # GH7405
+        for c, d in (np.zeros(5), np.zeros(5)), \
```

Comment:
can you specify dtypes to the ``np.zeros`` constructor? (otherwise you have a platform comparison issue)